terraform/helm/fullnode/templates/fullnode.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       - name: fullnode
         image: {{ .Values.image.repo }}:{{ .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
-        command: ["/opt/aptos/bin/aptos-node", "-f", "/opt/aptos/etc/fullnode.yaml"]
+        command: ["sh", "-c", "ulimit -n 1048576 && /opt/aptos/bin/aptos-node -f /opt/aptos/etc/fullnode.yaml"]
```

Comment:
Is there a way in k8s config itself to specify this?

Reply:
Not that I know of that's easy -- I think it's more clear this way to have it inline